tests/terraform/runner/resources/plan_and_tf_combine_graph_with_missing_resources/source/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
```

Comment:
I'm writing it here as I don't have a better place - 
Basically I understand the test because we talked about it, but it's hard to read it if you are new to this bug.
Could you specify which resources are missing from the plan? 
For example, you can call the `aws_s3_bucket` by the name `not-in-tf-plan` instead of `example` (assuming this resource is missing)